go.mod

```diff
@@ -2,10 +2,14 @@ module github.com/nitrictech/cli
 
 go 1.18
 
+replace github.com/nitrictech/nitric/core => ../nitric/core
```

Comment:
A branch reference to to deploy-contract branch would help sort out some of the CI/CD build issues.

currently on commit: `5614c4e59cd9f9b51f30dc6358857d2bdfc92e8e`